ui/components/data-sources/DataSourcesContainer.js

```diff
@@ -35,6 +35,12 @@ class DataSourcesContainer extends Component {
         this.setState({ filter: { name } });
     }
 
+    handleKeyPress(event) {
```

Comment:
I think this belongs to `CommonToolbar`. We want the filter to _never_ refresh the browser so the container (or any component rendering the toolbar) should not worry about this.